docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -177,6 +177,18 @@ Request Body:
 
   Not supported when `lb-provider=ovn` is configured in openstack-cloud-controller-manager.
 
+- `loadbalancer.openstack.org/health-monitor-delay`
+
+  Defines the health monitor delay in seconds for the loadbalancer pools.
+
+- `loadbalancer.openstack.org/health-monitor-timeout`
+
+  Defines the health monitor timeout in seconds for the loadbalancer pools. This value should be less than delay
```

Comment:
nit: better s/delay/value of loadbalancer.openstack.org/health-monitor-delay/ ?

Reply:
@jichenjc delay term is coming from OpenStack APIs and I think we should keep it in same terminology.